mantidproject/mantid
# ConvertHFIRSCDtoMDE zero count check for vanadium

**Description of work.**

This adds a check to `ConvertHFIRSCDtoMDE` to exclude zero count pixels during vanadium normalization which was causing a divide by zero error. This was causing problems using `HB3AFindPeaks` since no UB matrix would be found as a result.

**To test:**



Run the new unit test for `HB3AFindPeaks` to make sure peaks and a UB matrix are found when using a vanadium file with some zero count pixels.


*There is no associated issue.*






Version into `ornl-next` #30609 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
